src/client/api/events/sub_milestone_latest.c

```diff
@@ -0,0 +1,42 @@
+
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+#include "client/api/events/sub_milestone_latest.h"
+#include <stdlib.h>
+#include "client/api/json_utils.h"
+#include "client/network/mqtt/mqtt.h"
+
+milestone_latest_t *res_milestone_latest_new(void) {
+  milestone_latest_t *res = (milestone_latest_t *)malloc(sizeof(milestone_latest_t));
+  if (res == NULL) {
+    printf("[%s:%d] OOM\n", __func__, __LINE__);
+    return;
```

Comment:
NULL or `res` should be passed to the caller.
```suggestion
    return NULL;
```

Reply:
Done.